worker/src/RTC/RtpDictionaries/RtpParameters.cpp

```diff
@@ -105,11 +105,6 @@ namespace RTC
 		if (data->mid())
 		{
 			this->mid = data->mid()->str();
-
-			if (this->mid.empty())
-			{
-				MS_THROW_TYPE_ERROR("empty mid");
-			}
```

Comment:
This was a terrible idea. In the copy constructor we may perfectly copy from a `RtpParameters` instance whose `mid` was not set and hence it has empty string as value, and this block of code would literally throw. There is no reason for this check here. If `mid` is not set then it is an empty string in `RtpParameters`, but then, when moving it to Node/Rust, we avoid serializing it so we are good. This check was buggy and totally unneeded.